api/editor/api-editor/src/main/java/ca/bc/gov/tno/controllers/KafkaController.java

```diff
@@ -34,106 +34,107 @@
 @RestController
 @RequestMapping("/kafka")
 public class KafkaController {
-	@Value("${kafka.bootstrap.servers}")
-	String bootstrapServers;
+  @Value("${kafka.bootstrap.servers}")
+  String bootstrapServers;
 
-	/**
-	 * List all topics
-	 *
-	 * @return
-	 * @throws InterruptedException
-	 * @throws ExecutionException
-	 */
-	@GetMapping("/topics")
-	public Set<String> getTopics() throws InterruptedException, ExecutionException {
-		var props = new Properties();
-		props.put("bootstrap.servers", bootstrapServers);
-		props.put("connections.max.idle.ms", 10000);
-		props.put("request.timeout.ms", 5000);
+  /**
+   * List all topics
+   *
+   * @return
+   * @throws InterruptedException
+   * @throws ExecutionException
+   */
+  @GetMapping("/topics")
+  public Set<String> getTopics() throws InterruptedException, ExecutionException {
+    var props = new Properties();
+    props.put("bootstrap.servers", bootstrapServers);
+    props.put("connections.max.idle.ms", 10000);
+    props.put("request.timeout.ms", 5000);
 
-		try (var client = KafkaAdminClient.create(props)) {
-			var topics = client.listTopics();
-			var names = topics.names().get();
-			if (names.isEmpty()) {
-				// case: if no topic found.
-			}
-			return names;
-		} catch (InterruptedException | ExecutionException e) {
-			// Kafka is not available
-			throw e;
-		}
-	}
+    try (var client = KafkaAdminClient.create(props)) {
+      var topics = client.listTopics();
+      var names = topics.names().get();
+      if (names.isEmpty()) {
+        // case: if no topic found.
```

Comment:
Probably intentional, but was there something supposed to be thrown here?

Reply:
Just a PoC endpoint atm.  So I have no intention of completing the work until we need to.

Reply:
Gotchya!